src/mixedmodel.jl

```diff
@@ -15,6 +30,19 @@ Equality comparisons are used b/c small non-negative θ values are replaced by 0
 """
 issingular(m::MixedModel, θ=m.θ) = any(lowerbd(m) .== θ)
 
+# FIXME: better to base this on m.optsum.returnvalue
+StatsBase.isfitted(m::MixedModel) = m.optsum.feval > 0
+
+StatsBase.meanresponse(m::MixedModel) = mean(m.y)
+
+StatsBase.modelmatrix(m::MixedModel) = m.feterm.x
```

Comment:
With the new Xymat, the things that are non copies are explicitly views, right? Then it's pretty obvious based on the return type whether we're returning a copy or not. I'm wondering if we should document somewhere which of these return views, references and copies.

Reply:
Probably.  Do you have any suggestions of where "somewhere" should be?

Reply:
Perhaps in the docstring? Then it's right there, both in code and where users look first.

Reply:
Turns out that the `modelmatrix` method returns a stored matrix (`m.feterm.x`), not a view or a copy.  The `FeTerm` field contains the original model matrix from the `StatsModels` function and the `Xymat` field contains the augmented matrix with `y` in the last column.  If the model has been weighted then `Xymat` contains the original and the weighted `Xy` matrices.  For unweighted cases there are two references to the same matrix.